azure-mgmt-resources/src/test/java/com/azure/management/resources/core/InterceptorManager.java

```diff
@@ -37,7 +36,8 @@
 public class InterceptorManager {
 
     private final static String RECORD_FOLDER = "session-records/";
-    private static final String BODY_LOGGING = "x-ms-body-logging";
+    private final static String BODY_LOGGING = "x-ms-body-logging";
+    private final static Integer HTTP_TEMPORARY_REDIRECT = 307; // HTTP Status Code
```

Comment:
just int?

Reply:
The original one is int https://hc.apache.org/httpcomponents-core-4.4.x/httpcore/apidocs/org/apache/http/HttpStatus.html

Reply:
Got it. I mean just use `int` should be OK, no need for `Integer` (which could have problem when using ==).

Reply:
Got it, I have seen the `Integer.ParseInt`, so I thought it would get Integer before.